environment.py has been turned into a directory,
without actually porting the code using it.

--- a/sphinx/pycode/__init__.py
+++ b/sphinx/pycode/__init__.py
@@ -352,7 +352,7 @@
     import pprint
     x0 = time.time()
     # ma = ModuleAnalyzer.for_file(__file__.rstrip('c'), 'sphinx.builders.html')
-    ma = ModuleAnalyzer.for_file('sphinx/environment.py',
+    ma = ModuleAnalyzer.for_file('sphinx/environment/__init__.py',
                                  'sphinx.environment')
     ma.tokenize()
     x1 = time.time()
